Dockerfile-windows

```diff
@@ -11,6 +10,11 @@ LABEL org.opencontainers.image.authors="Consul Team <consul@hashicorp.com>" \
       org.opencontainers.image.title="consul" \
       org.opencontainers.image.description="Consul is a datacenter runtime that provides service discovery, configuration, and orchestration."
 
+# FROM envoyproxy/envoy-windows:v1.19.5 as envoy
+# COPY --from=envoy ["Program Files\envoy", "C:\envoy"]
+RUN mkdir C:\\envoy
+COPY build-support-windows/envoy.exe envoy/envoy.exe
```

Comment:
For now, we have to manually copy the `envoy.exe` to the `build-support-windows` folder